File: leetcode/q412/q412.cs
public class Solution {
    public IList<string> FizzBuzz(int n) {
        String[] answer = new String[n];

        for (int i = 1; i < n + 1; i++)
        {
            if (i % 3 == 0 && i % 5 == 0)
            {
                answer[i - 1] = "FizzBuzz";
            }
            else if (i % 3 == 0)
            {
                answer[i - 1] = "Fizz";
            }
            else if (i % 5 == 0)
            {
                answer[i - 1] = "Buzz";
            }           
            else
            {
                answer[i - 1] = "" + i + "";
            }
            
        }                   
        return answer;
    }
}

File: leetcode/q412/q412.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        answer: List[str] = []

        for i in range(1, n + 1):
            if (i % 15 == 0):
                answer.append("FizzBuzz")
            elif (i % 5 == 0):
                answer.append("Buzz")
            elif (i % 3 == 0):
                answer.append("Fizz")
            else:
                answer.append(f"{i}")
        
        return answer
        
            
        
